src/api.js

```diff
@@ -0,0 +1,4 @@
+const OMDB_API = 'http://www.omdbapi.com/?apikey=65c1586c&t=';
```

Comment:
```suggestion
const OMDB_API = 'https://www.omdbapi.com/?apikey=65c1586c&t=';
```
your page does't work because of it